Examples/S3EventNotifier/README.md

```diff
@@ -0,0 +1,79 @@
+# S3 Event Notifier
+
+This example demonstrates how to write a Lambda that is invoked by an event originating from Amazon S3, such as a new object being uploaded to a bucket.
+
+## Code
+
+In this example the lambda function receives an `S3Event` object defined in the `AWSLambdaEvents` library as input object. The `S3Event` object contains all the information about the S3 event that triggered the function, but what we are interested in is the bucket name and the object key, which are inside of a notification `Record`. The object contains an array of records, however since the lambda is triggered by a single event, we can safely assume that there is only one record in the array: the first one. Inside of this record, we can find the bucket name and the object key:
```

Comment:
the second occurence of 'lambda' must be replaced by 'Lambda function' (capital L and the words "Lambda function" together, not just Lambda)